internal/imported.go

```diff
@@ -27,6 +27,16 @@ func randomFillImportedFunc() extism.HostFunction {
 	return randomFillImported
 }
 
+// localOffset returns an Extism Function for retrieving the offset of the local time zone in seconds.
+func localOffsetImportedFunc() extism.HostFunction {
+	getOffsetFunc := extism.NewHostFunctionWithStack("local_offset_secs", func(ctx context.Context, p *extism.CurrentPlugin, stack []uint64) {
```

Comment:
I would use a more generic name for this function:

```suggestion
	getOffsetFunc := extism.NewHostFunctionWithStack("utc_offset_seconds", func(ctx context.Context, p *extism.CurrentPlugin, stack []uint64) {
```